wire/conn.go

```diff
@@ -5,7 +5,7 @@ import "github.com/zach-klippenstein/goadb/internal/errors"
 const (
 	// The official implementation of adb imposes an undocumented 255-byte limit
 	// on messages.
-	MaxMessageLength = 255
+	MaxMessageLength = 1024 * 1024
```

Comment:
Given the above comment, there should also be an explanation where this number comes from.

Reply:
Good point, I've just updated the comment. Also, I renamed `MaxMessageLength` to `MaxPayloadSize` for better reflecting adb's [implementation](https://cs.android.com/android/platform/superproject/+/master:packages/modules/adb/adb.h;drc=bea8f3c0203a77a114bf70885befc582f3f6cef6;l=33).